module/metrics/transaction.go

```diff
@@ -21,6 +21,9 @@ type TransactionCollector struct {
 	timeToExecuted             prometheus.Summary
 	timeToFinalizedExecuted    prometheus.Summary
 	transactionSubmission      *prometheus.CounterVec
+	executeScriptDuration      *prometheus.CounterVec
+	transactionResultDuration  *prometheus.CounterVec
```

Comment:
These should probably be `prometheus.Histogram` to track the value over time. Count is usually used for values where we want to track how often/how many of something occur.

e.g. https://github.com/onflow/flow-go/blob/master/module/metrics/execution.go#L302-L308